src/mote/config.py

```diff
@@ -0,0 +1,4 @@
+MEETING_DIR = "/srv/web/meetbot"
+MEETBOT_URL = "https://meetbot-raw.fedoraproject.org"
+DATAGREPPER_BASE_URL = "https://apps.fedoraproject.org"
+RECOGNIITION_PATTERN = "(.*)[\-\.]([0-9]{4}-[0-9]{2}-[0-9]{2})-([0-9]{2}\.[0-9]{2})"
```

Comment:
You should always use raw string for rexexp
```suggestion
RECOGNIITION_PATTERN = r"(.*)[\-\.]([0-9]{4}-[0-9]{2}-[0-9]{2})-([0-9]{2}\.[0-9]{2})"
```